### PR TITLE
Support to add task to specific category

### PR DESCRIPTION
**Pull request by danitetus**
If you add **categoryId** property to the options object, you can save the task to a specific category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ const tasks = [
     {title: 'from api tomorrow', dueDate: 'tomorrow'},
     {title: 'from api upcoming', dueDate: 'upcoming'},
     {title: 'from api someday', dueDate: null},
+    {title: 'from api to category', categoryId: 'youtCategoryId'}
```
Comment (davoam) [inline README.md:19]: typo: `yourCategoryId`

Comment (davoam) [reply README.md:19]: what will be if a user runs this code, but do not change `yourCategoryId` to a real category id? will it fail?

**Review by davoam — COMMENTED** [review]
Thanks for the PR!

**Comment by danitetus** [comment]
No, in my tests, if the category id was wrong, the task was saved into the default category (you can define wich is the default category on the app).

**Comment by davoam** [comment]
> No, in my tests, if the category id was wrong, the task was saved into the default category (you can define wich is the default category on the app).

got it, thanks. Let's add a js comment to this line about that, fix typo, and then merge

**Comment by danitetus** [comment]
I have added the js comment to that line and fixed the typo. After merge, could you update the npm package?

**Comment by davoam** [comment]
> Please could you merge the commits and update the npm package? I really need this función on one project.
> 
> Thanks

**Pull request merged**